### ISSUE TITLE
Move config parameters from class to instance

### ISSUE THREAD

**Issue by ikhomutov** [crowdin/crowdin-api-client-python#20]
Please make it possible to specify configuration parameters during instance initialization. Like you did for every other wrappers.
```python
client = CrowdinClient(token='__TOKEN__', organization='organizationName')
```
Your current approach looks very weird.

**Comment by andrii-bodnar** [comment]
HI @ikhomutov, thanks a lot for your suggestion!

**Comment by svenXY** [comment]
Wouldn't that just be a proper constructor method?

**Comment by andrii-bodnar** [comment]
@svenXY good idea. The main thing here is to save the backward compatibility without the need to increment the major version of the package

**Comment by svenXY** [comment]
It is certainly possible, but ugly...
Something along the lines of

```python

# this is not good code!
class CrowdinClient:
    API_REQUESTER_CLASS: Type[APIRequester] = APIRequester

    TIMEOUT = 60
    RETRY_DELAY = 0.1  # 100ms
    MAX_RETRIES = 5
    HTTP_PROTOCOL = "https"
    BASE_URL = "api.crowdin.com/api/v2/"
    HEADERS = {}
    USER_AGENT = "crowdin-api-client-python"
    PAGE_SIZE = 25

    def __init__(self, 
                 organization: str|None =None,
                 token: str=None,
                 base_url: str|None=None,
                 user_agent: str|None=None,
                 page_size: str|None=None,
                 timeout: int|None=None,
                 retry_delay: int|float|None=None,
                 max_retries: int|None=None,
                 http_protocol: str|None=None,
                 headers: dict|None=None
                 ):
        self.ORGANIZATION=organization or self.ORGANIZATION
        self.TOKEN=token or self.TOKEN,
        self.BASE_URL=base_url or self.BASE_URL,
        self.USER_AGENT=user_agent or self.USER_AGENT,
        self.PAGE_SIZE=page_size or self.PAGE_SIZE,
        self.TIMEOUT=timeout or self.TIMEOUT,
        self.RETRY_DELAY=retry_delay or self.RETRY_DELAY,
        self.MAX_RETRIES=max_retries or self.MAX_RETRIES,
        self.HTTP_PROTOCOL=http_protocol or self.HTTP_PROTOCOL,
        self.HEADERS=headers or self.HEADERS
        self._api_requestor = None

        if self.ORGANIZATION is None:
            self._platform_type = PlatformType.BASIC
        else:
            self._platform_type = PlatformType.ENTERPRISE
```

This allows for using params while at the same time keep the old way of subclassing CrowdinClient with those constants, but it means passing around the same stuff over and over. Also, those str|None etc. in the signature are ugly and kind of contradict the idea of typing.

I'd prefer to move the constants into the constructor, but that would break the documented way of subclassing with constants...

**Comment by andrii-bodnar** [comment]
@svenXY thank you! I would suggest removing the old way of initializing from the Readme. Also, please check the CI - some checks have failed on your PR

**Comment by svenXY** [comment]
It works now both ways and the tests succeed.
It's a bit kludgy, but maybe worth keeping it this way until the next major release is due and then remove the old way

**Comment by svenXY** [comment]
hi @andrii-bodnar it looks like this has not yet been released to pypi? Any reasons? Or - when can I hope to find it there?

**Comment by andrii-bodnar** [comment]
@svenXY not yet, I'm going to release the new version today in a few hours

**Comment by andrii-bodnar** [comment]
@svenXY the new version is already available - [1.6.0](https://pypi.org/project/crowdin-api-client/1.6.0/)
